### DISCUSSION TITLE
Use with rehypeRaw?

### DISCUSSION THREAD

**Discussion by CWengen** [bitekong/rehype-mermaid#2] [Q&A]
Hi! Very useful. I'm quite new to working with rehype/reactmarkdown etc.

I am trying to get this to work together with rehypeRaw, as I want to render both HTML and Mermaid from one markdown document. Is this possible?

![image](https://user-images.githubusercontent.com/27075997/132471777-64485125-5e9b-4ac7-9659-6ef7ef8d3ca1.png)

Currently, the mermaid graph never renders, unless I remove rehypeRaw.
Or is there anything else I could do, to make react markdown render HTML together with Mermaid?

**Comment by XXXXXXXXXXCode** [comment]
you could exchange the ordering of `rehypeRaw ` and `rehypeMermaid `. the reason of this phenomenon may be the result svg picture which `rehypeMermaid` rendered is considered as a plain html object by `rehypeRaw`.
here is a sample I pass.
```
const test = `
<a>test</a>
\`\`\`mermaid
graph TB
    a --> b
    b --> c

\`\`\`
`

...some other codes

<ReactMarkdown rehypePlugins={[rehypeRaw, rehypeMermaid]}>
    {test}
</ReactMarkdown>
```
